docs/UserGuide.md

```diff
@@ -157,11 +178,24 @@ Finds the staff/patients that used to be affiliated or are currently affiliated
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​
 * Adding/Editing a person's affiliation automatically updates the affiliation history for the person.
-* Deleting a person automatically deletes the person from others' affilation history.
 
 Examples:
 * `affnh 2` lists the people who used to be affiliated or are currently affiliated to the 2nd person in the contact list.
 
+### Remove affiliation history of a person: `removeah`
+
+Format: `removeah INDEX`
+
+Remove all affiliation history of patient/staff indicated by the given `INDEX`, except for affiliations that are currently affiliated.
+
+* Remove all affiliation history for the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index **must be a positive integer** 1, 2, 3, …​
+* Current affiliation in affiliation history will not be removed.
```

Comment:
Maybe something like "Does not remove the person they are currently affiliated to from affiliation history"

Reply:
Right! Thats sounds better will change it in the next commit!